cmd/start-amazon-cloudwatch-agent/path.go

```diff
@@ -33,7 +33,6 @@ func startAgent(writer io.WriteCloser) error {
 		execArgs := []string{
 			agentBinaryPath, // when using syscall.Exec, must pass binary name as args[0]
 			"-config", tomlConfigPath, "-envconfig", envConfigPath,
-			"-pidfile", AGENT_DIR_LINUX + "/var/amazon-cloudwatch-agent.pid",
```

Comment:
Though this seems to be safe, it still changes the existing behavior. If customers build their own image with cloudwatch agent and want to run it with other applications in one container, they might still want the pid file. Could you introduce an option to disable the generation of pid file?